### PR TITLE
[etherpad] Containerized the building script

### PR DESCRIPTION
**Pull request by pabloyoyoista**
Still missing in this pull request:
 - Update the FROM directive once the docker registry name for mafiasi has been decided
 - Update the README with new information

### CODE REVIEW

**Review by Akasch — APPROVED** [review]
Looks good.

registry is https://registry.mafiasi.de/

Maybe compare with https://github.com/ether/etherpad-lite/blob/develop/Dockerfile and

**Comment by Akasch** [comment]
Personally I would try to update to buster directly

**Comment by pabloyoyoista** [comment]
> Maybe compare with https://github.com/ether/etherpad-lite/blob/develop/Dockerfile and 
> Personally I would try to update to buster directly

This was a bit tricky because etherpad Dockerfile references node-10 Dockerfile. Debian buster has nodejs as a package, so decided to install directly from repos.

File: etherpad/Containerfile [hunk 1]
```diff
@@ -1,36 +1,38 @@
...
+						  gnupg \
+						  dirmngr \
+						  wget && \
+    apt-get install --yes --no-install-recommends nodejs npm
+
+RUN export buildDeps='git' && \
+    useradd --uid 2005 --create-home etherpad && \
+    apt-get install --yes --no-install-recommends $buildDeps && \
+    git clone https://github.com/ether/etherpad-lite.git /opt/etherpad && \
+    git --git-dir=/opt/etherpad/.git --work-tree=/opt/etherpad checkout ${VERSION} && \
```
Comment (pabloyoyoista) [inline etherpad/Containerfile:18]: Etherpad's Dockerfile copy all the contents of their git repo to the container. Pulling from github and checking out the desired version should get same results.

**Comment by lilioid** [comment]
etherpad is now already containerized althoug not via pull request.

**Pull request closed without merging**